### DISCUSSION TITLE
please somebody tell me how to set the colour of the lines / chart elements for different charts

### DISCUSSION THREAD

**Discussion by apostolos-geyer** [jtablesaw/tablesaw#955] [Q&A]
i must know 

with urgency

**Comment by apostolos-geyer** [comment]
> i must know
> 
> with urgency

it's not that urgent but i want to know

**Comment by lwhite1** [comment]
Look at the builders for Marker, and other builders, starting in the class
you’re using.

For more formal documentation, see the Plotly JavaScript docs, as
everything eventually is used to generate JS code that uses the plotly
library
